test/RequestTest.php

```diff
@@ -100,6 +101,20 @@ public function testWhitelistHeaders()
   }
 
   /**
+   * @expectedException Castle_CurlOptionError
+   */
+  public function testCastleCurlOptions()
+  {
+    // Will not throw.
+    Castle::setCurlOpts(array(CURLOPT_CONNECTTIMEOUT => 1,
+                              CURLOPT_CONNECTTIMEOUT_MS => 1000,
```

Comment:
We might want to set this to 2000 to be consistent with `castle-rb`. (Or update `castle-rb`)